exchanges/oex/oex.go

```diff
@@ -0,0 +1,501 @@
+package oex
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"net/url"
+	"sort"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/config"
+	exchange "github.com/thrasher-corp/gocryptotrader/exchanges"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/request"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/ticker"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/websocket/wshandler"
+	log "github.com/thrasher-corp/gocryptotrader/logger"
+)
+
+// Oex is the overarching type across this package
+type Oex struct {
+	exchange.Base
+	WebsocketConn *wshandler.WebsocketConnection
+}
+
+const (
+	oexAPIURL     = "https://openapi.oex.com"
+	oexAPIVersion = ""
```

Comment:
Please set this to v1

Reply:
OEX versioning is inconsistent across their API which is also why oexAPIVersion is not used anywhere currently

Example: 
```
/open/api/v2/all_order
```
and 
```
/open/api/all_trade
```

are both active and valid endpoints having just a set "oexAPIVersion" might cause some confusion?

Is it worth expanding to oexAPIv1 & oexAPIv2 and updating paths in the http request?